packages/default-icons/src/figma-icons/index.ts

```diff
@@ -158,3 +159,22 @@ export { default as TimelinePoint24 } from './TimelinePoint24';
 export { default as TimelinePoints24 } from './TimelinePoints24';
 export { default as Ninja24 } from './Ninja24';
 export { default as Kontur24 } from './Kontur24';
+export { default as Minus16 } from './Minus16';
+export { default as ZoomTo16 } from './ZoomTo16';
+export { default as North16 } from './North16';
+export { default as Ruler16 } from './Ruler16';
+export { default as Line16 } from './Line16';
+export { default as PointOutline16 } from './PointOutline16';
```

Comment:
need to fix all duplicates here, it will build then

Reply:
yeah i've found it too, thanks!
was too confused by the amount of changes to pay attention to it